jdaviz/core/template_mixin.py

```diff
@@ -1433,13 +1434,14 @@ def __init__(self, plugin, items, selected, viewer,
         self.update_wcs_only_filter(only_wcs_layers)
 
         self.filter_is_root = is_root
+        self.has_children = has_children
         self.filter_is_child_of = is_child_of
 
         if self.filter_is_root:
-            # ignore layers that are children in associations:
             def filter_is_root(data):
                 return self.app._get_assoc_data_parent(data.label) is None
 
+            # ignore layers that are children in associations:
             self.add_filter(filter_is_root)
```

Comment:
Just a note to self: I think this really needs to be refactored so that filters are passed directly to the init rather than all these boolean flags, but that can be follow-up since its not being introduced here (I also noticed this while working on #2827)